docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/libraries/charts/library/bypass-appropriations.js

```diff
@@ -1,16 +1,18 @@
 const dataFileName = 'bypass-appropriations';
 const id = 'NCI-Chart__appropriations';
 
-function initChart(Chart, appropriationsData) {
+function initChart(Chart, chartData) {
+  const { data } = chartData;
+
   new Chart(id, {
-    chart: appropriationsData.chartType,
-    title: appropriationsData.chartTitle,
-    subtitle: appropriationsData.chartSubTitle,
-    xAxis: appropriationsData.xAxis,
-    plotOptions: appropriationsData.plotOptions,
-    yAxis: appropriationsData.yAxis,
-    series: appropriationsData.series,
-    tooltip: appropriationsData.tooltip
+    chart: data.chartType,
+    title: data.chartTitle,
+    subtitle: data.chartSubTitle,
+    xAxis: data.xAxis,
+    plotOptions: data.plotOptions,
+    yAxis: data.yAxis,
+    series: data.series,
+    tooltip: data.tooltip
```

Comment:
This change is unrelated to the bug fix.  Please undo it.

Reply:
I'll partially retract that.  This is actually part of the overall pattern of passing extra data and refactoring all the charts to accommodate it.

Reply:
This goes to the same issue you mentioned above which I addressed in the earlier reply.

Reply:
@blairlearn Reverted changes made to the other chart files. Now just passing in new argument to function for `grants-contracts` chart.